frontend/packages/console-dynamic-plugin-sdk/scripts/package-definitions.ts

```diff
@@ -66,7 +66,6 @@ export const getCorePackage: GetPackageDefinition = (
   manifest: {
     name: '@openshift-console/dynamic-plugin-sdk',
     version: sdkPackage.version,
-    type: 'module',
```

Comment:
This was causing webpack 5 to resolve ESM module requests more strictly.

https://webpack.js.org/guides/ecma-script-modules/

> Imports in ESM are resolved more strictly. Relative requests must include a filename and file extension.

For now, omitting the `type` field in package manifest. (This impacts imports in SDK source code.)